Sources/Identity/SubscriptionInfo.swift

```diff
@@ -75,6 +75,9 @@ import Foundation
     /// Paid price for the subscription
     @objc public let price: ProductPaidPrice?
 
+    /// Management purchase URL
+    @_spi(Internal) public let managementURL: URL?
```

Comment:
`managementURL`, or `managementUrl`?

Reply:
Well, it's `managementURL` in `CustomerInfo`, but it's `customerUrl` in the models that are decoded (automatically snake cased) 🙃 
I've followed that convention